k8s/README.md

```diff
@@ -28,3 +28,55 @@ While port forwarding, access to the port of local machine.
 http://localhost:8080
 ```
 
+# Shared `.kettle` and `.pentaho` directory
+
+Pods share `.kettle` and `.pentaho` directory as PersistentVolumeClaim (PVC).
+If you want to deploy these PVCs to a Kubernetes cluster with multiple nodes, you can configure YAML files so that they can be shared among multiple nodes.
+
+Edit `kettle-pvc.yaml` and `pentaho-pvc.yaml`.
+
+```yaml
+  # - ReadWriteOnce # Comment out
+  - ReadWriteMany # Uncomment
```

Comment:
Can we make `ReadWriteMany` by default?
From end-user perspective, `~/.kettle` and `~/.pentaho` should be the same no matter which container/node happens to serve them.
Let me know if you have other thoughts that I'm missing.

Reply:
I think it is OK to set `ReadWriteMany` as default. In case that users have to change it to `ReadWriteOnce`, I modified README.md like this
```
Pods share `~/.kettle` and `~/.pentaho` directory as PersistentVolumeClaim (PVC).
If you want to deploy these PVCs to a Kubernetes cluster which only supports `ReadWriteOnce`, you can configure YAML files.
```

Reply:
Thank you for updating.
For a record, not all "Volume Plugin" supports `ReadWriteMany` (https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).